test_data/lockdownd.log

```diff
@@ -0,0 +1,159 @@
+10/13/21 07:57:42.865446 pid=69 mglog: libMobileGestalt MGBasebandSupport.c:183: No IMEI in CT mobile equipment info dictionary
```

Comment:
Could we please rename the test data file ios_lockdownd.log? This way it's clearer what it's for.